domains/_vercel.yean.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+      "username": "joshex150",
+      "email": "joshex150@gmail.com"
+    },
+    "record": {
+      "TXT": "vc-domain-verify=yean.is-a.dev,c671dbf711ab7400712a"
```

Comment:
Assuming this is for a vercel domain, but I'm not seeing a file with yean.json in the domains folder or this PR, make sure you have that included as well.

Reply:
I've done this a couple of times now, what exactly should I include in the second file

Reply:
Preferably the actual A/CNAME records that Vercel wants. Currently you're just requesting a verification, but that's useless if the site is not being pointed towards Vercel.

Refer to this guide for further details: https://www.is-a.dev/docs/vercel/

Reply:
I understand what to do now, thank you!